.github/workflows/check-dist.yml

```diff
@@ -23,10 +23,10 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Set Node.js 12.x
+      - name: Set Node.js 16
         uses: actions/setup-node@v2.4.1
         with:
-          node-version: 12.x
+          node-version: 16.16.0
```

Comment:
Wouldn't it be better to use `16.x` so that it picks the latest available version at the time?